### PR TITLE
tcti: fix null ptr dereference in tcti_tbs_receive

### PR DESCRIPTION
**Pull request by tstruk**
If response_size is NULL the function will dereference a NULL ptr before checking.

### CODE REVIEW

File: src/tss2-tcti/tcti-tbs.c [hunk 1]
```diff
@@ -325,4 +327,4 @@ const TSS2_TCTI_INFO*
 Tss2_Tcti_Info (void)
 {
     return &tss2_tcti_info;
-}
\ No newline at end of file
+}
```
Comment (williamcroberts) [inline src/tss2-tcti/tcti-tbs.c]: Nit: superfluous white space change, split to another patch.

Comment (tstruk) [reply src/tss2-tcti/tcti-tbs.c]: There is no change. This is just the git way of noting that the file does not ends with '\n'
It looks different in gitub GUI. The git diff in terminal looks like this:
```
@@ -325,4 +327,4 @@ const TSS2_TCTI_INFO*
 Tss2_Tcti_Info (void)
 {
     return &tss2_tcti_info;
-}
\ No newline at end of file
+}
```
See https://mrkiffie.com/2016/newlines-and-version-control/

Comment (williamcroberts) [reply src/tss2-tcti/tcti-tbs.c]: Freaking UI's
